package.json

```diff
@@ -30,6 +30,7 @@
     "@react-aria/interactions": "3.5.0",
     "@react-spectrum/utils": "3.6.1",
     "@react-stately/toggle": "3.2.2",
+    "@sentry/nextjs": "^6.12.0",
```

Comment:
This should be a devDependency right? Should havet exact versions too, `-E` flag

Reply:
Not sure on this, the closed PR before this didn't save it that way so went the same direction.